src/github.com/getlantern/lantern-ui/app/_css/app.css

```diff
@@ -7602,7 +7602,7 @@ body[dir=rtl] #latestDetails dd {
 #vis-container, #vis {
   height: 100%;
   background-color: #fff;
-  overflow: scroll;
+  overflow: hidden;
```

Comment:
It's not clear to me what this change for. User may not see the full content via a not maximized window, which is very common.

We can reduce the `margin-bottom` or `padding-bottom` of `#sites` to eliminate extra spaces.